pa11ycrawler/pipelines.py

```diff
@@ -223,7 +232,12 @@ def process_item(self, item, spider):
                 )
             )
 
-        self.check_title_match(item['page_title'], stdout, spider.logger)
+        if stdout:
+            pa11y_results = json.loads(stdout.decode('utf8'))
```

Comment:
ah, perhaps this is the decode from higher up? though, I suppose I'm still curious about decoding the title.